lib/compiler/aro/aro/Compilation.zig

```diff
@@ -308,7 +308,7 @@ fn generateSystemDefines(comp: *Compilation, w: anytype) !void {
         ),
         else => {},
     }
-    if (comp.target.abi == .android) {
+    if (comp.target.isAndroid()) {
```

Comment:
@Vexu @ehaas just a heads up for this Aro hunk; it's easy to miss since it won't be a compile error.